browser-extension/common/src/custom-elements/test-rule-widget/index.css

```diff
@@ -5,7 +5,7 @@
   right: 16px;
 }
 
-#container {
+#rq-container {
```

Comment:
Why we made these changes to id names? The CSS is specific to this shadow DOM, it wont have any conflicts